README.md

```diff
@@ -47,9 +52,22 @@ The `scope` option is used to set up a global
 for all events and the
 [`client`](http://getsentry.github.io/sentry-javascript/interfaces/node.nodeoptions.html) option
 is used as a Sentry instance or to initialize an internally used Sentry instance.
+
 The internally used client (initialized in either way) is accessible through
 `server.plugins['hapi-sentry'].client`.
 
+## Own Sentry instance
+
+You can pass a `Sentry` instance to  the `client` option if you already initialized your own.
+Example:
```

Comment:
one more line-break? To make this 'heading' show up on a new line in  the rendered markdown

Reply:
i knew how it would render. do you think it's better on a new line?